appveyor.yml

```diff
@@ -31,10 +35,13 @@ install:
 build_script:
   - cmd: |
       set PATH=%GOPATH%/bin;%PATH%
-      mkdir -p $GOPATH\bin
-      go build -o $GOPATH\bin\nomad
-# test_script:
-#   - cmd: gotestsum -f short-verbose --junitfile results.xml
+      mkdir -p %GOPATH%\bin
+      go build -o %GOPATH%\bin\nomad.exe
+test_script:
+  # only test docker driver tests for now
+  - cmd:
+      gotestsum --junitfile results.xml
+           github.com/hashicorp/nomad/drivers/docker
```

Comment:
We should move this and dependencies installation to external scripts.  Should we use bat files?  Anything equivalent to Makefiles?

Reply:
We could actually use the makefile, I wanted to avoid doing a big refactor to make everything x-plat for expediency though.